core/src/main/javascript/io/Url.js

```diff
@@ -317,12 +321,13 @@
                                     throw this._exception;
                                 }
 
-                                // this allows the stream to return already-parsed JSON
+                                // This allows the stream to return
+                                // already-parsed JSON.
                                 if (result.response.json !== undefined) {
                                     this._json = result.response.json;
                                     this.getJSON = function () { return self._json; };
```

Comment:
Adding this function here caused memory problems - somehow things were not GC'd as expected.